docs/parts/prism_central.md

```diff
@@ -0,0 +1,108 @@
+
+### 2. Set up Virtual IP, ISCSI IP, and NTP for the Cluster
+
+Prism Central requires an iSCSI IP address for the cluster. This address is used to provide storage to the Prism Central service.
+
+In the next part we'll be installing a service that requires the cluster have a virtual IP as well as NTP configured, so we'll be configuring that now as well.
+
+Click on the gear icon in the upper right corner of the Prism UI. Then choose `Cluster Details` and enter `192.168.103.254` for the Virtual IP and `192.168.103.253` for the ISCSI Data Services IP and and click `Save`.
+
+![Cluster Details](../images/iscsi-ip.png)
+
+We also need to configure NTP for the cluster. Click on the gear icon in the upper right corner of the Prism UI and select `NTP Servers` from the settings menu on the left. Then add a local NTP pool server for the region you deployed Nutanix into. We'll use `0.north-america.pool.ntp.org`
+
+![NTP Servers](../images/ntp-servers.png)
+
+### 3. Deploy Prism Central
+
+Next, choose `Prism Central Registration` and select `Register or create new`.
+
+![Prism Central Registration](../images/prism-central-deploy.png)
+
+Choose `Deploy`, pick whatever version it offers you and click `Next`.
+
+![prism-central-version](../images/prism-central-version.png)
+
+and then `Deploy Single-VM PC` and click `Next`.
+
+![Single VM](../images/single-vm.png)
+
+Then choose `Small` and choose `Create Network` from the `Network` drop down.
+
+![Small Deployment](../images/small-deployment.png)
+
+Enter `VM Network` and VLAN ID `0` and click `Save`.
+
+![VM Network](../images/vm-network.png)
+
+Now enter `255.255.252.0` for the Subnet Mask and `192.168.100.2` for the Gateway, and `192.168.103.252` for the VM IP and click `Next`.
+
+![VM More Network](../images/vm-more-network.png)
+
+Now be patient while Prism Central installs. This can take a while. When finished, you'll be back at the `Register or create new` screen again. That's normal. Let's verify Prism Central is installed by checking the VM tab and looking for the Prism Central VM. This screenshot has highlighted ways to identify the VM.
+
+![Prism Central VM Tab](../images/prism-central-vm-tab.png)
+
+### 4. Register the Prism Central VM
+
+Now that Prism Central is deployed, we need to register the cluster to it. Click on the gear icon in the upper right corner of the Prism UI and select `Prism Central Registration`. Click `Register or create new` and then click `Connect` under `I already have a Prism Central instance deployed`.
+
+![Prism Central Registration](../images/prism-central-registration.png)
+
+Click `Next` on the `Connect Info` page and then put in the Prism Central IP address `192.168.103.252`, port of `9440`, and the username and password you used to log in to Prism. This will be the default Prism password. Click `Next`.
```

Comment:
Is this supposed to be the new password I entered when I was forced to change the password?  Or the original default password from the Nutanix docs?

Reply:
It's supposed to be the original default Prism password.

Reply:
I've added a new workflow that has you update the passwrod on prism central and THEN register prism to it.